src/elements/content-uploader/Footer.js

```diff
@@ -23,6 +23,10 @@ type Props = {
 };
 
 const Footer = ({ isLoading, hasFiles, errorCode, onCancel, onClose, onUpload, fileLimit, isDone }: Props) => {
+    const isCloseButtonDisabled = hasFiles;
+    const isCancelButtonDisabled = !hasFiles || isDone;
+    const isUploadButtonDisabled = !hasFiles;
```

Comment:
moved the closeButton and uploadButton disabled booleans up as well, for readability. Not necessary though

Reply:
i dig it